src/standalone/Soil/soil_albedo.jl

```diff
@@ -0,0 +1,200 @@
+with water content).
+"""
+struct ConstantTwoBandSoilAlbedo{
+    SF <: Union{AbstractFloat, ClimaCore.Fields.Field},
+} <: AbstractSoilAlbedoParameterization
+    "Soil PAR Albedo"
+    PAR_albedo::SF
+    "Soil NIR Albedo"
+    NIR_albedo::SF
+end
+
+"""
+    update_albedo!(bc::AtmosDrivenFluxBC, albedo::ConstantTwoBandSoilAlbedo, p, soil_domain, model_parameters)
+
+Updates PAR and NIR albedo using the constant parameters provided in `albedo`.
+"""
+function update_albedo!(
+    bc::AtmosDrivenFluxBC,
+    albedo::ConstantTwoBandSoilAlbedo,
+    p,
+    soil_domain,
+    model_parameters,
+)
+    @. p.soil.PAR_albedo = albedo.PAR_albedo
+    @. p.soil.NIR_albedo = albedo.NIR_albedo
+end
+
+"""
+     CLMTwoBandSoilAlbedo{
+        FT <: AbstractFloat,
+        SF <: Union{FT, ClimaCore.Fields.Field},
+    } <: AbstractSoilAlbedoParameterization
+
+A parameterization for soil albedo: the soil albedo is 
+defined in two bands (PAR and NIR), can spatially vary or
+be set to scalar, and varies with water content at the surface
+θ_sfc, according to CLM:
+
+α = min(α_wet + Δ(θ_sfc), α_dry), where
+Δ(θ_sfc) = max(0.11 - 0.40*θ_sfc,0).
+
+We use a value for θ_sfc averaged over the depth `albedo_calc_top_thickness`.
+If the model resolution is such that the first layer is thicker than this depth,
+the value from the first layer is used.
+
+CLM reference: Lawrence, P.J., and Chase, T.N. 2007. Representing a MODIS consistent land surface in the Community Land Model
+(CLM 3.0). J. Geophys. Res. 112:G01023. DOI:10.1029/2006JG000168.
+"""
+struct CLMTwoBandSoilAlbedo{
+    FT <: AbstractFloat,
+    SF <: Union{FT, ClimaCore.Fields.Field},
+} <: AbstractSoilAlbedoParameterization
+    "Soil PAR Albedo dry"
+    PAR_albedo_dry::SF
+    "Soil NIR Albedo dry"
+    NIR_albedo_dry::SF
+    "Soil PAR Albedo wet"
+    PAR_albedo_wet::SF
+    "Soil NIR Albedo wet"
+    NIR_albedo_wet::SF
+    "Thickness of top of soil used in albedo calculations (m)"
+    albedo_calc_top_thickness::FT
+end
+
+function CLMTwoBandSoilAlbedo{FT}(;
+    PAR_albedo_dry,
+    NIR_albedo_dry,
+    PAR_albedo_wet,
+    NIR_albedo_wet,
+    albedo_calc_top_thickness = FT(0.02),
+) where {FT}
+    return CLMTwoBandSoilAlbedo{FT, typeof(PAR_albedo_dry)}(
+        PAR_albedo_dry,
+        NIR_albedo_dry,
+        PAR_albedo_wet,
+        NIR_albedo_wet,
+        albedo_calc_top_thickness,
+    )
+end
+
+
+"""
+    albedo_from_moisture(θ_sfc::FT, albedo_dry::FT, albedo_wet::FT)
+
+Calculates pointwise albedo for any band as a function of soil surface moisture given
+the dry and wet albedo values for that band.
+
+CLM reference: Lawrence, P.J., and Chase, T.N. 2007. Representing a MODIS consistent land surface in the Community Land Model
+(CLM 3.0). J. Geophys. Res. 112:G01023. DOI:10.1029/2006JG000168.
+"""
+function albedo_from_moisture(
+    θ_sfc::FT,
+    albedo_dry::FT,
+    albedo_wet::FT,
+) where {FT}
+    Δ = max(FT(0.11) - FT(0.4) * θ_sfc, FT(0))
```

Comment:
It would be good to note where 0.11 and 0.4 come from here (maybe they're just the values used by CLM?)